packages/core/src/loop/workflows/agentic-loop/index.ts

```diff
@@ -0,0 +1,151 @@
+import type { StepResult, ToolSet } from 'ai-v5';
```

Comment:
This workflow used to get created inside of `stream.ts` but I moved it out to better organize the code.